### PR TITLE
[Fix] Fix failed lint workflow

### PR DESCRIPTION
**Pull request by LazyBusyYang**
1. Fix failed lint workflow, caused by an old PR.
2. Add class RGB2BGR and function rgb2bgr, literally fix some color conversion.
3. Add function auto_load_smpl_data, choose a correct class when you forget of which type the npz file is.

### CODE REVIEW

File: tests/transform/image/test_color.py [hunk 1]
```diff
@@ -13,6 +13,9 @@ def test_bgr2rgb():
     bgr_image = bgr2rgb(rgb_image, color_dim=0)
```
Comment (yl-1993) [inline tests/transform/image/test_color.py:13]: `rgb_image = bgr2rgb(bgr_image, color_dim=0)`?

File: xrmocap/transform/image/color.py [hunk 2]
```diff
@@ -66,8 +87,40 @@ def bgr2rgb(input_array: Union[np.ndarray, torch.Tensor],
...
         b_backup = input_array[tuple(b_slice_list)].copy()
-    input_array[tuple(b_slice_list)] = input_array[tuple(r_slice_list)]
-    input_array[tuple(r_slice_list)] = b_backup
-    return input_array
+        if not inplace:
+            ret_array = input_array.copy()
+        else:
+            ret_array = input_array
+    ret_array[tuple(b_slice_list)] = input_array[tuple(r_slice_list)]
+    ret_array[tuple(r_slice_list)] = b_backup
```
Comment (yl-1993) [inline xrmocap/transform/image/color.py:101]: Save `tuple(b_slice_list)` and `tuple(r_slice_list)` to two variables in adavance? It seems that they have been used serveral times.

File: tests/data_structure/body_model/test_smplxd_data.py [hunk 3]
```diff
@@ -181,3 +181,7 @@ def test_file_io():
     with pytest.raises(FileExistsError):
         smplxd_data.dump(npz_path, overwrite=False)
     smplxd_data.dump(npz_path, overwrite=True)
+    # test auto load
+    instance, class_name = auto_load_smpl_data(npz_path)
+    assert isinstance(instance, SMPLXDData)
```
Comment (yl-1993) [inline tests/data_structure/body_model/test_smplxd_data.py:186]: Check the instance shape by the way?

File: xrmocap/transform/image/color.py [hunk 4]
```diff
@@ -66,8 +87,40 @@ def bgr2rgb(input_array: Union[np.ndarray, torch.Tensor],
...
+            Which dim is the color channel. Defaults to -1.
+        inplace (bool, optional):
+            Whether it is an in-place operation.
+            Defaults to False.
+
+    Returns:
+        Union[np.ndarray, torch.Tensor]:
+            Same type as the input.
+    """
+    return bgr2rgb(
```
[the remark this answers is not in the document]
Comment (LazyBusyYang) [reply xrmocap/transform/image/color.py]: Fixed in [36c845f](https://github.com/openxrlab/xrmocap/pull/78/commits/36c845f7f7918df025d92827c556e9892a88ddce).

File: tests/transform/image/test_color.py (hunk 1 above)
Comment (LazyBusyYang) [reply tests/transform/image/test_color.py]: Fixed in [36c845f](https://github.com/openxrlab/xrmocap/pull/78/commits/36c845f7f7918df025d92827c556e9892a88ddce).

File: tests/data_structure/body_model/test_smplxd_data.py (hunk 3 above)
Comment (LazyBusyYang) [reply tests/data_structure/body_model/test_smplxd_data.py:186]: Fixed in [36c845f](https://github.com/openxrlab/xrmocap/pull/78/commits/36c845f7f7918df025d92827c556e9892a88ddce).

File: xrmocap/transform/image/color.py (hunk 2 above)
Comment (LazyBusyYang) [reply xrmocap/transform/image/color.py]: Fixed in [db0b34d](https://github.com/openxrlab/xrmocap/pull/78/commits/db0b34dfe3f5b5f8de5992a88bff7b99c5cd21ea).

**Review by yl-1993 — APPROVED** [review]

**Pull request merged**